### PR TITLE
Fix bug in DeleteInterviewCommand

### PR DESCRIPTION
**Pull request by ghost**
When deleting an interview, the applicant who was originally scheduled for that interview still appeared as if they had an Interview scheduled in the ApplicantList.
Now deleting an interview correctly sets the related Applicant's 'hasInterview' to false.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteInterviewCommand.java [hunk 1]
```diff
@@ -42,7 +45,9 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         Interview interviewToDelete = lastShownList.get(targetIndex.getZeroBased());
-        model.deleteInterview(interviewToDelete);
+        Applicant applicantToUpdate = interviewToDelete.getInterviewApplicant();
+        model.setApplicant(applicantToUpdate, applicantToUpdate.getApplicantWithoutInterview());
```
Comment (jingjie88) [inline src/main/java/seedu/address/logic/commands/DeleteInterviewCommand.java]: Maybe the set applicant can be put inside the deleteInterview method in ModelManager.

**Review by jingjie88 — APPROVED** [review]
Overall LGTM!

File: src/main/java/seedu/address/logic/commands/DeleteInterviewCommand.java (hunk 1 above)
Comment (ghost) [reply src/main/java/seedu/address/logic/commands/DeleteInterviewCommand.java]: Fixed, thanks for the review

**Review by jingjie88 — APPROVED** [review]
LGTM!

**Pull request merged**